### ISSUE TITLE
Error for installation

### ISSUE THREAD

**Issue by MohammadAmin94** [googlemaps/google-maps-services-java#286]
i try add that codes to gradle in android studio and when i run my app this error will be shown:

> Error:Error converting bytecode to dex:
> Cause: Dex cannot parse version 52 byte code.
> This is caused by library dependencies that have been compiled using Java 8 or above.
> If you are using the 'java' gradle plugin in a library submodule add 
> targetCompatibility = '1.7'
> sourceCompatibility = '1.7'
> 
i looking for solution like [this](https://stackoverflow.com/a/37294741/4474316) but not working
plz help me how fix that

**Comment by apjanke** [comment]
What version of Android Studio are you using, and what Android SDK version are you targeting in your project?

I tried to reproduce this in Android Studio 2.3.3, doing a minimal test app targeting API 15 (Ice Cream Sandwich). But the build failed with a dependency version conflict.

```
/Users/janke/AndroidStudioProjects/TestAppToReproGoogleMapsApiVersionError/app/build.gradle
Error:Conflict with dependency 'com.google.code.findbugs:jsr305' in project ':app'. Resolved versions for app (1.3.9) and test app (2.0.1) differ. See http://g.co/androidstudio/app-test-app-conflict for details.
```

Using `./gradlew app:dependencies`, I tracked it down to what looks like a conflict between the espresso-core and google-guava dependencies.

```
androidTestCompile - Classpath for compiling the androidTest sources.
\--- com.android.support.test.espresso:espresso-core:2.2.2
     +--- com.google.code.findbugs:jsr305:2.0.1

compile - Classpath for compiling the main sources.
+--- com.google.maps:google-maps-services:0.2.0
     \--- com.google.guava:guava:22.0
          +--- com.google.code.findbugs:jsr305:1.3.9
```

Sounds like Guava is using a rather old version of `com.google.code.findbugs:jsr305`.

If someone could help me sort this out, I may be able to help reproduce and debug this.

**Comment by domesticmouse** [comment]
FWIW, you really shouldn't be using `google-maps-services-java` in Android, as it opens you up to the threat of your API key being stolen.

**Comment by domesticmouse** [comment]
This will be fixed in v0.2.2
